openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMSWIG.java

```diff
@@ -88,16 +76,17 @@
      * @throws ModelLoadingException in case any LightGBM error occurs.
      */
     LightGBMSWIG(final String modelPath, final DatasetSchema schema) throws ModelLoadingException {
+
+        final String LightGBMParameters = "num_threads=1";
```

Comment:
If one day I want to add more parameters, like [device_type]( https://lightgbm.readthedocs.io/en/latest/Parameters.html#device_type), how could I do that?
is it "num_threads=1, device_type=cpu"?
I think we should change this in order to be flexible for further changes

Reply:
Yes it is :) A single string. I don't think we have parameters to use there.
What do you suggest changing? Or given that it's just a string it's good?

Reply:
I think we can define it within the class, but out of the constructor as a private final static String and maybe explain it on javadoc that we can add more parameters by adding comma.
WDYT?

Reply:
Hmm I like the idea but if you really want to modify this, it would likely be passed as a parameter to the constructor so each model could use its custom parameters.

Making it static would go against it.

Reply:
I think it would be better then.
Then you will have to create the variable on the constructor of LightGBMBinaryClassificationModel, but this time it can be a static variable right?